python/GangaPanda/Lib/Panda/Panda.py

```diff
@@ -134,6 +148,16 @@ def convertDQ2NamesToQueueName(locations):
 
 def convertQueueNameToDQ2Names(queue):
 
+    ret=[pandaresources[x]['ddm'] for x in pandaresources if pandaresources[x]['name']==queue]
+    # logger.warning('(%s) ret:%s:' % (type(ret), ret))
+    try:
+        ret=ret[0]
+    except:
+        # logger.warning('oh well')
+        import traceback
+        logger.error('queue %s error %s'%(queue, str(traceback.print_exc() )))
+    return str(ret).split(',')
+
```

Comment:
Is the previous way of converting Queue name -> DDM name now completely dead? i.e. is the code following this based around:
```
    for site in Client.PandaSites[queue]['setokens'].values():
        sites.append(Client.convSrmV2ID(site))
```
broken now? I ask because if it is I'll just remove it now :smile: